package/MDAnalysis/coordinates/TXYZ.py

```diff
@@ -0,0 +1,159 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- http://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+"""
+TXYZ file format --- :mod:`MDAnalysis.coordinates.TXYZ`
+=======================================================
+
+Coordinate reader for Tinker xyz files .txyz and trajectory .arc files
+Differences between Tinker format and normal xyz files: 
+ - there is only one header line containing both the number of atoms and a comment
+ - column 1 contains atom numbers (starting from 1)
+ - column 6 contains atoms types 
+ - the following columns indicate connectivity (atoms to which that particular atom is bonded, according to numbering in column 1)
+
+Tinker format: http://chembytes.wikidot.com/tnk-tut00#toc2
+Tinker: https://dasher.wustl.edu/tinker/
+
+"""
+import numpy as np
+import os
+from __future__ import (absolute_import, division)
```

Comment:
Your tests fail because 
```
SyntaxError: from __future__ imports must occur at the beginning of the file
```
Fix this in your files and then check that your tests pass.